client/components/like-button/button.jsx

```diff
@@ -83,8 +83,14 @@ var LikeButton = React.createClass( {
 
 		containerClasses = classnames( containerClasses );
 
+		if ( likeCount === 0 && ! this.props.showCount ) {
+			likeCount = '';
+		} else {
+			likeCount = likeCount + ' ';
```

Comment:
Maybe I'm missing some of the story - we can still add right padding to an inline element right? I just feel like ' ' is akin to adding &nbsp; for spacing...